README.md

```diff
@@ -60,3 +60,4 @@ Environment variables descriptions:
     * `extensions` - retrieves the list of other users (contacts)
   * `HTTP_CLIENT_SSL_VERIFY_TYPE` - configures the SSL verification type for HTTP client requests that utilize HTTPS schema; acceptable values include: `verify_none`, `verify_peer`
     (optional, default value is `verify_peer`)
+  * `HIDE_BALANCE_IN_USER_INFO` - controls whether to hide balance in user info (optional, default value is `false`)
```

Comment:
This environment variable is clearly related to PortaBilling functionality. I suggest adding the `PORTABILLING_` prefix to it, resulting in `PORTABILLING_HIDE_BALANCE_IN_USER_INFO`. Additionally, it should be placed after the `PORTABILLING_FILTER_CONTACTS_WITHOUT_EXTENSION` environment variable, both in the documentation and in the source code processing order, for consistency.